app-misc/recoll/recoll-1.31.0.ebuild

```diff
@@ -0,0 +1,131 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..9} )
```

Comment:
Could you update this? The current active python supports are 3.9 and 3.10. Nothing bad having 3.7-3.8 in there either, but 3.6 is a bit...

Reply:
3.7 is the same as 3.6 at this point, it's not a valid target, i.e. can't select it, it's just dead code -- removing 7 whenever possible will save mgorny some work later when cleaning these up in a `*/*` commit

So check if 10 works, if it does then `3_{8..10}`, if it doesn't, then `3_{8..9}`